addons/base_vat/base_vat.py

```diff
@@ -359,4 +363,57 @@ def check_vat_tr(self, vat):
 
         return False
 
+    def check_vat_nl(self, vat):
+        """
+        Temporary Netherlands VAT validation to support the new format introduced in January 2020,
+        until upstream is fixed.
+
+        Algorithm detail: http://kleineondernemer.nl/index.php/nieuw-btw-identificatienummer-vanaf-1-januari-2020-voor-eenmanszaken
+
+        TODO: remove when fixed upstream
+        """
+
+        try:
+            from stdnum.util import clean
+            from stdnum.nl.bsn import checksum
+        except ImportError:
+            return True
+
+        vat = "NL" + vat
```

Comment:
Can you add a comment that this is needed up until Odoo 10.0 (because Odoo 11.0 and up contain this commit: https://github.com/odoo/odoo/commit/f643bca6cf2e).

Reply:
Done!

Reply:
Thanks! The VAT validation is a bit of a mess after that commit so I've created https://github.com/odoo/odoo/pull/45095. It does not apply to Odoo 8.0